week2-problems/problem-2.3.2/problem-2.3.2.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"log"
+)
+
+/*
+Given 2 slices of ints, a and b, return true if they have the same first element, or they have the same last element. Both slices will be length 1 or more.
+commonEnd([1, 2, 3], [7, 3]) → true
+commonEnd([1, 2, 3], [7, 3, 2]) → false
+commonEnd([1, 2, 3], [1, 3]) → true
+*/
+
+func twoSlicesOfSameLengthChecker(sliceOne []int, sliceTwo []int) bool {
+	if len(sliceOne) == len(sliceTwo) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func firstOrLastElementChecker(sliceOne []int, sliceTwo []int) bool {
+	lastIndex := len(sliceOne) - 1
```

Comment:
in example we have two slices with any elements, so you need to look at two lengths